src/OrleansServiceBus/Providers/Streams/EventHub/EventHubCheckpointer.cs

```diff
@@ -22,16 +19,16 @@ internal class EventHubPartitionCheckpoint
         private Task inProgressSave;
         private DateTime? throttleSavesUntilUtc;
 
-        public bool Exists { get { return entity != null && entity.Offset != EventHubConsumerGroup.StartOfStream; } }
+        public bool Exists => entity != null && entity.Offset != EventHubConsumerGroup.StartOfStream;
```

Comment:
be careful, this is a c# 6 declaration, and out VSO doesn't build this yet... I recommend you revert it for now